inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_proposal_to_proposal_ie.cpp

```diff
@@ -8,58 +8,78 @@
 #include <vector>
 
 #include <ngraph/opsets/opset1.hpp>
-
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
 #include <ngraph_ops/proposal_ie.hpp>
 #include <ngraph/rt_info.hpp>
 
-ngraph::pass::ConvertProposalToLegacyMatcher::ConvertProposalToLegacyMatcher() {
-    auto input_0 = std::make_shared<pattern::op::Label>(element::f32, Shape{1, 1, 1, 1});
-    auto input_1 = std::make_shared<pattern::op::Label>(element::f32, Shape{1, 1, 1, 1});
-    auto input_2 = std::make_shared<pattern::op::Label>(element::f32, Shape{3});
+bool convert_to_proposal_ie(std::shared_ptr<ngraph::op::v0::Proposal> proposal, bool add_second_out = false) {
+    ngraph::Output<ngraph::Node> last;
```

Comment:
Do not use Output without particular node instance.

Reply:
I leaved this declaration but I do not use undeclared `last`. Now `last `is set explicitly either to reshape node or node from where im_info is coming with acceptable shape. Please review.